bug-1674187-pref-holdback-study-for-quan.toml

```diff
@@ -0,0 +1,59 @@
+[metrics]
+daily = ["active_hours", "searchmode_counts", "searchalias_counts", "searchmode_amazon_counts", "searchalias_amazon_counts", "searchoneoff_amazon_counts"]
+weekly = ["ad_clicks", "search_count", "organic_search_count", "tagged_search_count", "tagged_follow_on_search_count"]
+overall = ["active_hours", "uri_count", "ad_clicks", "search_count"]
```

Comment:
This config file will extend the default set instead of replacing it; you get all of `"active_hours", "uri_count", "ad_clicks", "search_count"` overall for free. The default config is here: https://github.com/mozilla/jetstream/blob/main/jetstream/config/default_metrics.toml